atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,64 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF{
+    private ListaEncadeada top;
+    private int size;
+
+    public PilhaComLista() {
+        top = null;
+        size = 0;
+    }
+
+    @Override
+    public void push(Integer element) throws Exception {
+        if (size > 10) {
```

Comment:
Apenas um ponto de atenção, cuidado com magic number ou valores constantes soltos pelo código, o ideal é levar esse valor para uma variável que represente o contexto daquela informação, por exemplo:

primeiro declara a variável constante com um nome que descreve o valor que ela receber
```
private final int MAX_ELEMENT = 10;
```

```suggestion
        if (size > MAX_ELEMENT) {
```